content/sql/concepts/math-functions/terms/sin/sin.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'SIN()'
+Description: 'Returns the sine of an angle.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'SQL'
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'Math Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`SIN()`** function in SQL is a mathematical function that returns the sine of an angle. It is compatible with various SQL database systems such as MySQL, PostgreSQL, Oracle, and SQL Server.
+
+## Syntax
+
+```sql
```

Comment:
I'd like the code block to be a ` ```pseudo` block with just the function. Putting it this way implies it's only valid in the `SELECT` clause. Follow the code block with a short sentence explaining what `angle` is.

Reply:
Hey @SSwiniarski, can you pls explain to me a little bit more because I don't understand your change request?